pipeline/steps/az-ext-oms.yaml

```diff
@@ -20,36 +23,43 @@ steps:
         $rgName="$env:WS_RG"
         $workspaceName="$env:WS_NAME"
         $workspaceSubscriptionName="$env:WS_SUB_NAME"
+        $env = "${{ parameters.env }}"
 
         az account set -s $workspaceSubscriptionName
-
-        $key=az monitor log-analytics workspace get-shared-keys --resource-group $rgName --workspace-name $workspaceName --query "primarySharedKey" -o tsv
-        $id=az monitor log-analytics workspace show --resource-group $rgName --workspace-name $workspaceName -o tsv --query "customerId"
-
-        Write-Host "ID: $id"
-
-        $settings = [PSCustomObject]@{
-          workspaceId           = $id
+                                                    
+        $environmentObj = [PSCustomObject]@{
+            "sbox" = "sandbox"
+            "stg"  = "staging"
+            "prod" = "production"
```

Comment:
Could these be variables instead of a mapping in the object?
so in the variables-sbox.yaml have a new var env_long=sandbox, then pass this in

Reply:
Changed to variables in the variables-<env>.yaml file